benchmarks/problems/wfg_problem.py

```diff
@@ -0,0 +1,86 @@
+import math
```

Comment:
Cannot we directly use `wfg.py` or fix it to use with kurobako? I think the codes of this file are redundant.